modules/vmseries/main.tf

```diff
@@ -34,12 +34,13 @@ data "aws_kms_alias" "current_arn" {
 resource "aws_network_interface" "this" {
   for_each = var.interfaces
 
-  subnet_id         = each.value.subnet_id
-  private_ips       = lookup(each.value, "private_ips", null)
-  source_dest_check = lookup(each.value, "source_dest_check", false)
-  security_groups   = lookup(each.value, "security_group_ids", null)
-  description       = lookup(each.value, "description", null)
-  tags              = merge(var.tags, { Name = coalesce(try(each.value.name, null), "${var.name}-${each.key}") })
+  subnet_id          = each.value.subnet_id
+  private_ips        = lookup(each.value, "private_ips", null)
+  source_dest_check  = lookup(each.value, "source_dest_check", false)
+  security_groups    = lookup(each.value, "security_group_ids", null)
+  description        = lookup(each.value, "description", null)
+  tags               = merge(var.tags, { Name = coalesce(try(each.value.name, null), "${var.name}-${each.key}") })
+  ipv6_address_count = try(each.value.ipv6_address_count, null)

```

Comment:
What do you think about adding possibility to define specific IPv6 address using attribute [ipv6_addresses](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/network_interface#ipv6_addresses) ?

Reply:
I think this is great idea and the plan was to add a basic support for now (leveraging AWS-assigned IPv6 addresses) and add the support for specific IPv6 at next iteration. Or do you think we should add this functionality already?

Reply:
It's not must have :)  but let's consult that subject on planning meeting, ok ?

Reply:
After the discussion from the meeting we will keep it simple at this stage. It will be revisited during AWS modules refactor.